lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -253,6 +253,16 @@
             version: 6.6
           - name: "php"
             version: "7.0"
+    
+    - name: Create a stage deployment slot for an existing web app
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mywebapp/slots/stage
+        plan:
+          resource_group: myresourcegroup
+          name: myappplan
+        app_settings:
+          testkey:testvalue      
```

Comment:
Other check failed because of trailing white space here. I believe that to also be invalid.

Reply:
well, sanity tests are very strict about trailing whitespace, so you'll have to remove it